src/main/java/org/craftercms/studio/controller/rest/v2/DependencyController.java

```diff
@@ -68,4 +69,17 @@ public ResponseBody getSoftDependencies(@RequestBody @Valid GetSoftDependenciesR
         responseBody.setResult(result);
         return responseBody;
     }
+
+    @GetMapping(DEPENDENT_ITEMS)
+    public ResponseBody getDependentItems(@NotEmpty @ValidSiteId @RequestParam String siteId,
```

Comment:
What is the purpose of `ResponseBody`? It would simplify a bit to just return the `ResultOne`

Reply:
Fixed to use `ResultOne`. Thank you!